manuscript/chapter2-philosophy.md

```diff
@@ -99,12 +99,18 @@ Concluding, don't forget to take responsibility in case you are the culprit (by
 
 ## The virtuosity of reports
 
-Let’s think for a moment to all the rants or requests we receive every day, to how much better they would be if submitted in a **constructive way**. Gorgeous, isn’t it? Let’s take the first difficult step ourselves, let’s be ready to take it because we know how it’s powerful, and we know how to take it, so we can spread it to all the world and in our daily life, family, job and so on.  
+Let’s think for a moment to all the rants or requests we receive every day, to how much better they would be if submitted in a **constructive way**. 
+
+![](images/2/feedback.jpg)
+
+Gorgeous, isn’t it? Let’s take the first difficult step ourselves, let’s be ready to take it because we know how it’s powerful, and we know how to take it, so we can spread it to all the world and in our daily life, family, job and so on.  
```

Comment:
I'm stumbling about the „we know how it's powerful” part here.
If I expand it to „we know how it is powerful” I wonder what „it“ refers to.
But „we know how its powerful” is referring to some „it” as well.